.pre-commit-config.yaml

```diff
@@ -51,7 +51,7 @@ repos:
     - id: pip-from-conda
       additional_dependencies: [pyyaml]
       entry: python scripts/generate_pip_deps_from_conda.py
-      files: ^conda-envs/
+      files: ^conda-envs/environment-dev-py37\.yml$
```

Comment:
It's probably a good idea to have some CI jobs run with different versions of packages, to check PyMC3 works both with the oldest and with the newest versions of its dependencies. So, requirements-dev.txt now has to be generated from just one yml file